openapi/paths/getters/getAsset.json

```diff
@@ -17,6 +17,22 @@
           "type": "integer",
           "format": "int64"
         }
+      },
+      {
+        "name": "heightStart",
+        "in": "query",
+        "description": "Define the start height for trading information over a given period - if not given the last 24h are taken as trading period",
```

Comment:
maybe -> Define the start height for trading information over a given period - if not given the last  360 blocks (around 24hrs) are taken as trading period",